config/garden.json

```diff
@@ -0,0 +1,15 @@
+{
+  "mainnet": {
+    "TODO": "REPLACE THIS WITH REAL REGISTRY ONCE DEPLOYED!!!!",
```

Comment:
Please do not forget to update this. Consider keeping this comment as a reminder until resolved.